### PR TITLE
OBPIH-2871 Add PO number column to shipment items for stock movement …

### PR DESCRIPTION
**Pull request by DominikaHatala**
…from PO

### CODE REVIEW

File: src/groovy/org/pih/warehouse/api/StockMovementItem.groovy [hunk 1]
```diff
@@ -102,6 +102,7 @@ class StockMovementItem {
     }
 
     Map toJson() {
+        ShipmentItem shipmentItem = ShipmentItem.get(id);
```
Comment (jmiranda) [inline src/groovy/org/pih/warehouse/api/StockMovementItem.groovy:105]: We should pass the order number into the stock movement item when it's created. Look to see how the orderItemId is set and do the same.

Comment (DominikaHatala) [reply src/groovy/org/pih/warehouse/api/StockMovementItem.groovy]: @jmiranda I have changed this one.

File: grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -1257,6 +1257,7 @@ order.orderFrom.label=Ordered From
 order.orderHasBeenPlacedWithVendor.message=Purchase order {0} has been placed
 order.orderItems.label=Line Items
 order.orderNumber.label=Order Number
+order.purchaseOrderNumber.label=PO Number
```
Comment (jmiranda) [inline grails-app/i18n/messages.properties:1260]: Sorry I missed this earlier. Please just use order.orderNumber.label. If/when PIH wants to change the text, they can do so using a localization entry.

File: grails-app/i18n/messages.properties [hunk 3]
```diff
@@ -2580,6 +2581,7 @@ react.stockMovement.confirmPreviousPage.continue.label=Continue (lose unsaved wo
 react.stockMovement.message.confirmSubmit.label=Confirm submit
 react.stockMovement.confirmSubmit.message=Please confirm you are ready to submit your request. Once submitted, you cannot edit the request.
 react.stockMovement.confirmPickRefresh.message=This button will redo the autopick on all items that have not been previously edited. Are you sure you want to continue?
+react.stockMovement.orderNumber.label=PO number
```
Comment (jmiranda) [inline grails-app/i18n/messages.properties:2584]: Same here. Please call this Order Number.

Comment (jmiranda) [reply grails-app/i18n/messages.properties:2584]: We should really try to consolidate these i.e. `react.stockMovement` and `stockMovement` should be merged into `stockMovement`. But we can save that for another day.

**Comment by jmiranda** [comment]
@awalkowiak Can you review this when you get a chance?

**Comment by awalkowiak** [comment]
@jmiranda LGTM

**Pull request merged**